nixos/tests/ceph-multi-node.nix

```
@@ -198,6 +202,31 @@ let
         "ceph osd pool delete multi-node-other-test multi-node-other-test --yes-i-really-really-mean-it",
     )
 
+    # Add a metadata service according to https://docs.ceph.com/en/latest/cephfs/add-remove-mds/
+    monA.succeed(
+        "mkdir -p /var/lib/ceph/mds/ceph-${cfg.monA.name}",
+        "ceph auth get-or-create mds.${cfg.monA.name} mon 'profile mds' mgr 'profile mds' mds 'allow *' osd 'allow *' > /var/lib/ceph/mds/ceph-${cfg.monA.name}/keyring",
+        "chown -R ceph:ceph /var/lib/ceph/mds",
+        # Run ceph-mds once to connect it to a mon, otherwise it errors out. Are we missing [mon.N] sections in ceph.conf?
+        #   ceph-mds[881]: failed to fetch mon config (--no-mon-config to skip)
+        #"ceph-mds --cluster ceph -i ${cfg.monA.name} -m ${cfg.monA.ip}:6789",
+        "systemctl start ceph-mds.target",
+    )
+    monA.wait_for_unit("ceph-mds@${cfg.monA.name}.service")
```

Small typo:

```suggestion
    monA.wait_for_unit("ceph-mds-${cfg.monA.name}.service")
```

It should work now. :)

---

Thanks, applied!